speed-dreams/source-2.2.3/data/data/menu/DeveloperMenu.xml

```diff
@@ -147,6 +147,29 @@
             <attstr name="pushed image" val="data/img/button-left-pushed.png"/>
         </section>
 
+
+        <!-- Default button-->
+        <section name="DefaultButton">
+	        <attstr name="type" val="text button"/>
+	        <attstr name="show box" val="no"/>
+	        <attstr name="text" val="Default"/>
+	        <attstr name="tip" val="set default value for current interventiontype"/>
+	        <attstr name="h align" val="left"/>
+	        <attnum name="x" val="450"/>
+	        <attnum name="y" val="290"/>
```

Comment:
Move button down for a better spread of UI elements

Reply:
lower and more to the left